### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by arpit01923**
Alert component created

### CODE REVIEW

File: a [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* [32mAlert[m
+  Avatar[m
+  Badge[m
+  dev[m
+  main[m
```
Comment (rohanmathur91) [inline a]: what are this ?

**Review by rohanmathur91 — APPROVED** [review]
LGTM.

**Review by surbhi84 — CHANGES_REQUESTED** [review]
Add your html files as well, that'll help your peers to review your code better.
It is for future reference as well.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    --avatar-error: red;
+    --avatar-success: green;
+    --avatar-warning: orange;
+
+    --white-section: white;
+}
+
+body{
+    font-size: 62.5%;
+}
```
Comment (bajajcodes) [inline css/style.css]: Add new line here.

File: css/components/Alert.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    color: var(--avatar-error);
+}
+
+.outline-success{
+    color: var(--avatar-success);
+}
+
+.outline-warning{
+    color: var(--avatar-warning);
+}
```
Comment (bajajcodes) [inline css/components/Alert.css]: Add new line here.

File: a (hunk 1 above)
Comment (bajajcodes) [inline a]: What these lines of code mean?

**Review by bajajcodes — CHANGES_REQUESTED** [review]
Hey add video or image, to visualize what your are trying to communicate.

File: a (hunk 1 above)
Comment (arpit01923) [reply a]: just unnecessary file

Comment (arpit01923) [reply a]: just unnecessary link

**Comment by arpit01923** [comment]
image

**Pull request merged**